modules/lightning_features/lightning_api/modules/api_test/tests/src/Functional/ApiTest.php

```diff
@@ -138,94 +97,12 @@ public function testNotAllowed() {
     }
 
     // Cannot get unauthorized data anonymously.
-    $url = $this->buildUrl($this->paths['page_unpublished_get']);
+    $client = \Drupal::httpClient();
```

Comment:
Nitpick -- should be $this->container->get('http_client').

Reply:
✔️